### PR TITLE
Final var 2

### PR DESCRIPTION
**Pull request by Abhiram-Gullapalli**
#### Related issues

Part of issue #16662 

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by Abhiram-Gullapalli** [comment]
If I did something weird, please let me know if there are any anomalies.

File: files/en-us/web/javascript/guide/functions/index.md [hunk 1]
```diff
@@ -47,14 +47,19 @@ function myFunc(theObject) {
   theObject.make = 'Toyota';
 }
 
-var mycar = {make: 'Honda', model: 'Accord', year: 1998};
-var x, y;
+const mycar = {
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/functions/index.md:50]: I don't know what exactly is going on but this is giving me a feeling of deja vu...

File: files/en-us/web/javascript/guide/functions/index.md [hunk 2]
```diff
@@ -47,14 +47,19 @@ function myFunc(theObject) {
   theObject.make = 'Toyota';
```
Comment (teoli2003) [inline files/en-us/web/javascript/guide/functions/index.md:47]: Can you make a separate PR for this one? I think it deserves its own separate review. Thanks.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/javascript/guide/functions/index.md (hunk 2 above)
Comment (teoli2003) [reply files/en-us/web/javascript/guide/functions/index.md:47]: I had the impression of having already read this… Thank you.

Comment (teoli2003) [reply files/en-us/web/javascript/guide/functions/index.md:47]: @Abhiram-Gullapalli For each PR, you need to create a new branch from _main_ (not necessarily up-to-date), else you get extra files in the PR.
